.github/workflows/bcachefs-test.yml

```
@@ -0,0 +1,35 @@
+name: bchfs-tests
+
+on: [push]
+
+jobs:
+  bchfs-compile:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+          fetch-depth: 0
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+
+      - name: Install Requirements
+        run: |
+          git submodule update --init
+          pip install -r requirements.txt
+          pip install -r tests/requirements.txt
+
+      - name: Install
+        run: |
+          python setup.py -coverage install
+
+      - name: Tests
+        run: |
+          pytest tests
+          bash <(curl -s https://codecov.io/bash)
```

There was a bug in codecov recently:
https://about.codecov.io/security-update/

Something like the following might be a better way to run the cov
```suggestion
          curl -s -o codecov https://codecov.io/bash \
              && VERSION=$(grep 'VERSION=\".*\"' codecov | cut -d'"' -f2) \
              && shasum -a 512 -c --ignore-missing <(curl -s https://raw.githubusercontent.com/codecov/codecov-bash/${VERSION}/SHA512SUM)
          bash codecov
```